<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="OF060J7VPJ40X6N" failures="1" tests="11" name="Personal Banking" time="0.103" errors="0" timestamp="2025-05-22T10:18:39 CST">
  <testcase classname="test.TestNGDay1" name="demo1" time="0.012"/>
  <testcase classname="test.TestNGDay1" name="demo2" time="0.003"/>
  <testcase classname="test.TestNGDay1" name="methodForTestFail" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.TestNGDay1.methodForTestFail(TestNGDay1.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- methodForTestFail -->
  <testcase classname="test.TestNGDay1" name="parameterTest" time="0.001"/>
  <testcase classname="test.TestNGDay1" name="pickDataProvider" time="0.003"/>
  <testcase classname="test.TestNGDay1" name="pickDataProvider" time="0.001"/>
  <testcase classname="test.TestNGDay1" name="pickDataProvider" time="0.001"/>
  <testcase classname="test.TestNGDay1C2" name="apiLoginbank" time="0.001"/>
  <testcase classname="test.TestNGDay1C2" name="mobLoginbank" time="0.0"/>
  <testcase classname="test.TestNGDay1C2" name="parameterTest" time="0.001"/>
  <testcase classname="test.TestNGDay1C2" name="webLoginBank" time="0.0"/>
</testsuite> <!-- Personal Banking -->
